cypress/e2e/awx/access/credential-types/credentialTypes.cy.ts

```diff
@@ -137,7 +137,7 @@ describe('Credential Types', () => {
 
     it('can create a new credential type with no configs', () => {
       cy.createAndDeleteCustomAWXCredentialTypeUI(customCredentialTypeName);
-      cy.verifyPageTitle('Credential Types');
+      cy.verifyPageTitle('Create Types');
```

Comment:
sorry i had a typo! this is `Credential Types`

Reply:
no worries - just updated!